docs/source/notebooks/general/other_nuts_samplers.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>maybe pull out the constant kwargs in order to emphasize the nuts_sampler and minimize moving parts. </p><p>i.e. </p><p><code></code>`python</p><p>kwargs = {</p><p>    "draws": 2000, </p><p>    "chains": 5, </p><p>    "target_accept": 0.9, </p><p>    "random_seed": 42, </p><p>}</p><p> </p><p>idata_numpyro = model.fit(nuts_sampler="numpyro", **kwargs)</p><p>idata_blackjax = model.fit(nuts_sampler="blackjax", **kwargs)</p><p>...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>yes!</p>

Reply:
"its so easy to do 😄 "

Reply:
<p>Yes!</p>